be/src/formats/parquet/file_reader.cpp

```diff
@@ -168,9 +168,11 @@ Status FileReader::_read_min_max_chunk(const tparquet::RowGroup& row_group, vect
                 column_order = column_idx < column_orders.size() ? &column_orders[column_idx] : nullptr;
             }
 
-            Status status = _decode_min_max_column(*field, ctx.timezone, slot->type(), *column_meta, column_order,
-                                                   &(*min_chunk)->columns()[i], &(*max_chunk)->columns()[i]);
-            if (!status.ok()) {
+            bool decode_ok = false;
+            RETURN_IF_ERROR(_decode_min_max_column(*field, ctx.timezone, slot->type(), *column_meta, column_order,
+                                                   &(*min_chunk)->columns()[i], &(*max_chunk)->columns()[i],
+                                                   &decode_ok));
+            if (!decode_ok) {
```

Comment:
try the best to use min/max filter?

Reply:
> @dirtysalt I think you should explain the reason why this PR will fix the crash.

It's just because we ignore return status

```
RETURN_IF_ERROR(ColumnConverterFactory::create_converter(field, type, timezone, &converter));
```

The logic is the same to before: nothing changes, just check return status.

Reply:
Got you, thanks